lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -215,6 +215,37 @@ def provision
       end
 
       @logger.notify 'Spent %.2f seconds tagging VMs' % (Time.now - start)
+
+      # add additional disk to vm
+      @logger.notify 'Looking for disks to add...'
+
+
+      @hosts.each do |h|
+        hostname = h['vmhostname'].split(".")[0]
+
+        if h['disks']
+          @logger.debug "Found disks for #{hostname}!"
+          disks = h['disks']
+          disks.map!.with_index { |disk, index| "#{disk}_#{index}" }
+
+          disks.each do |disk|
```

Comment:
It seems like instead of doing a `.map!.with_index` and then a `.each` on that, you could do
```ruby
disks.each_with_index do |disk_size, index|
  #...
```
and it would be more obvious what's going on. Then:
```ruby
def disk_added?(host, disk_size, disk_index)
```
and
```ruby
def wait_for_disk(host, disk_size, disk_index)
   disk = "#{disk_size}_#{disk_index}"
```